CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 - [In Progress: 08 Apr 2021] Add automatic performance profiling for the Room database queries
 
 ### Changes
-- [In Progress: 12 Apr 2021] Show remove overdue appointment reasons in a separate screen
+- Show remove overdue appointment reasons in a separate screen
```

Comment:
Technically, this should be a separate commit.

Reply:
Ah, I think I accidentally combine it with the last commit. My bad 😅